components/resizable-iframe/index.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Imported from Calypso, with some lint fixes and gutenburg specific changes.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import debugFactory from 'debug';
+import { omit } from 'lodash';
+import uuid from 'uuid/v4';
+
+/**
+ * Globals
+ */
+const debug = debugFactory( 'gutenburg:resizable-iframe' ),
+	noop = () => {};
+
+export default React.createClass( {
+	displayName: 'ResizableIframe',
+
+	propTypes: {
+		src: React.PropTypes.string,
+		width: React.PropTypes.oneOfType( [
+			React.PropTypes.string,
+			React.PropTypes.number
+		] ),
+		height: React.PropTypes.oneOfType( [
+			React.PropTypes.string,
+			React.PropTypes.number
+		] ),
+		onLoad: React.PropTypes.func,
+		onResize: React.PropTypes.func,
+		title: React.PropTypes.string
+	},
+
+	getInitialState: function() {
+		return { width: 0, height: 0 };
+	},
+
+	getDefaultProps: function() {
+		return {
+			onLoad: noop,
+			onResize: noop,
+			title: uuid()
```

Comment:
Ah, that's a confusing rule description:

>`<iframe>` elements must have a unique title property to indicate its content to the user.

https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/iframe-has-title.md

The value of the rule is not in ensuring guaranteed uniqueness, but rather that users of screen readers are easily able to identify and distinguish the contents of the iframe. 

I expect the ESLint rule won't enforce its uniqueness, only that it's present.

To the purpose of the rule, I'd say we should set the title from the Twitter block as something like "Twitter Tweet embed". With that in mind, `ResizableIframe` should probably accept `title` as a prop and apply it to the rendered `iframe`.

The referenced guideline has some more useful context: https://dequeuniversity.com/rules/axe/1.1/frame-title